### ISSUE TITLE
Ensure Page Titles Are Descriptive and Informative

### ISSUE THREAD

**Issue by SamiyahKey** [cisagov/manage.get.gov#3176]
### Story

As a user navigating the website,
I need each page to have a clear and descriptive title,
So that I can quickly understand its purpose and content without unnecessary confusion.

https://getgov-staging.app.cloud.gov/admin/analytics/#content-start

### Updated Acceptance Criteria
1. Standardized Page Title Format
All page titles must follow the format:
[Page Title] | Django Admin
The following pages must be updated accordingly:

### Pages Requiring Title Updates:
- [x] Allowed Emails | Django Admin
- [x]  Domain Groups | Django Admin
- [x]  Domains | Django Admin
- [x]  Federal Agencies | Django Admin
- [x]  Hosts | Django Admin
- [x]  Portfolios | Django Admin
- [x]  Public Contacts | Django Admin
- [x]  Senior Officials | Django Admin
- [x]  Suborganizations | Django Admin
- [x]  Transition Domains | Django Admin
- [x]  User Domain Roles | Django Admin
- [x]  User Portfolio Permissions | Django Admin
- [x]  Users | Django Admin
- [x]  Verified by Staff | Django Admin
- [x]  Waffle Flags | Django Admin
- [x]  Websites | Django Admin
- [x]  Registrar Analytics | Django Admin

### Additional Context

- [ ] If a page’s content is user-specific or data-driven, ensure the title updates dynamically to reflect the current state.
- [ ] Verify that the < title > element inside the < head > is correctly updated on all pages.
- [ ] Use browser developer tools or automated accessibility testing tools (Lighthouse, Axe, ANDI) to confirm compliance with WCAG 2.4.2 Page Titled.
- [ ] Ensure clarity and consistency across all titles.
- [ ] Avoid vague or overly technical terms that may be unclear to users.

**Comment by katypies** [comment]
For this, @SamiyahKey , can you create a spreadsheet for each page that needs a title update, with the URL + title that it should be? Or provide a table in this ticket with that list? I wouldn't expect for the devs to necessarily know what all the titles should be, so we should be explicit about the titles that need to be updated.

**Comment by Katherine-Osos** [comment]
@katypies It looks like most pages in the admin already follow this pattern, so devs would just need to match the existing pattern when they find an inconsistency. Design can certainly try to find all those instances, but that would need to become its own separate ticket. That would be a lot of manual work on the design side where as  dev could find these titles programmatically. 

Should @SamiyahKey create a separate design ticket that blocks this one to find all the instances and create a document on what needs to change?
